astropy/io/votable/table.py

```diff
@@ -49,12 +53,17 @@ def parse(source, columns=None, invalid='exception', pedantic=None,
             - 'mask': mask out invalid values
 
     pedantic : bool, optional
-        When `True`, raise an error when the file violates the spec,
-        otherwise issue a warning.  Warnings may be controlled using
-        the standard Python mechanisms.  See the `warnings`
-        module in the Python standard library for more information.
-        When not provided, uses the configuration setting
-        ``astropy.io.votable.pedantic``, which defaults to False.
+        This is now an alias for ``verify='warn'`` (if ``pedantic`` is `False`)
+        or ``verify='exception'`` (if ``pedantic`` is `True`). The ``pedantic``
+        option will be deprecated in future.
+
+    verify : {'ignore', 'warn', 'exception'}, optional
+        When ``'exception'``, raise an error when the file violates the spec,
+        otherwise either issue a warning (``'warn'``) or silently continue
+        (``'ignore'``). Warnings may be controlled using the standard Python
+        mechanisms.  See the `warnings` module in the Python standard library
+        for more information. When not provided, uses the configuration setting
+        ``astropy.io.votable.verify``, which defaults to 'ignore'.
```

Comment:
Need to add a note of `.. versionchanged::`

Reply:
Fixed